core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -21,6 +21,7 @@ import java.net.URLEncoder
 import java.util.Date
 import javax.servlet.http.HttpServletRequest
 
+import scala.collection.mutable
```

Comment:
Can you directly import mutable.HashMap here? That's more consistent with style elsewhere